qmk_api_tasks.py

```diff
@@ -243,11 +247,11 @@ def run(self):
                         continue
 
                     if metadata.get('keymaps') and 'default' in metadata['keymaps']:
-                        keymap = fetch_json(metadata['keymaps']['default']['url'])
+                        keymap = fetch_json(metadata['keymaps']['default']['url'], as_hjson=True)
                         keymap['keymap'] = 'default'
                     else:
                         keymap_url = f'{KEYMAP_JSON_URL}/{keyboard[0]}/{keyboard.replace("/", "_")}_default.json'
-                        keymap = fetch_json(keymap_url)
+                        keymap = fetch_json(keymap_url, as_hjson=True)
```

Comment:
Should we expect the Configurator keymap to be JSON only?
```suggestion
                        keymap = fetch_json(keymap_url)
```

Reply:
If the cli sends hjson to the API then I guess it's acceptable. Configurator itself doesn't understand hjson and since it's not a widely used standard in js I would expect it to be adopted outside of cli.

Reply:
From the FAQ
```
"Hjson is a bad idea!"

Do not use it for

protocols
databases
Something like a REST service should never use Hjson. If you are writing a REST debugger you can convert to/from Hjson when the data is viewed by a human.
```

Reply:
Basically, you won't be able to debug it on the browser anymore as it's an opaque string.

Reply:
This isn't really to do with the API or the CLI, this code is pulling info.json and keymap.json directly from the qmk_firmware (and configurator) repo. Does that fall under "REST service", I wonder? Should we be asking the API for this data instead? Does API even have default keymap JSON?

Reply:
My concern here is the files are named JSON. This is seems to be but true. No JSON tool can process them. This leads to confusion like this. If the files were hjson this would be clear and obvious.

Reply:
VSCode config files are non-conforming and have comments too, fwiw. They still call them `*.json`.